pkg/server/sandbox_status_test.go

```diff
@@ -35,7 +35,10 @@ import (
 
 // Variables used in the following test.
 
-const sandboxStatusTestID = "test-id"
+const (
+	sandboxStatusTestID = "test-id"
+	testIP              = "10.10.10.10"
```

Comment:
`s/testIP/sandboxStatusTestIP`

Use a longer name to avoid conflict with other global variables in unit test in the same package.